### PR TITLE
Update all modules

### PR DESCRIPTION
**Pull request by jfy133**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/test_nothing.config [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 params {
     config_profile_name        = 'Test profile'
-    config_profile_description = 'Minimal test dataset without performing any preprocessing nor profiling to check pipeline function. Useful when you only wish to test a single profiler without having to 'opt-out' of all the others'
+    config_profile_description = "Minimal test dataset without performing any preprocessing nor profiling to check pipeline function. Useful when you only wish to test a single profiler without having to 'opt-out' of all the others"
```
Comment (Midnighter) [inline conf/test_nothing.config:15]: You're changing this differently in the other PR. Better to settle on one solution.

**Review by Midnighter — APPROVED** [review]
Looks like the changes to `taxprofiler.nf` are not technically related to updating modules but the whole thing looks good to me. Note that you're fixing the `test_nothing.config` in two different ways.

**Comment by jfy133** [comment]
> Looks like the changes to taxprofiler.nf are not technically related to updating modules

They are unfortunately, the module updates are breaking-changes in some cases, so I'm retrying to fix them.

Currently stuck on fastp bugging out with piping though :(

**Comment by jfy133** [comment]
OK fixed the fastp issue, hopefully this will work properly now @Midnighter are you OK with keeping the fancy new patch for now?

**Comment by Midnighter** [comment]
> are you OK with keeping the fancy new patch for now?

What exactly do you mean by the fancy new patch? 😛 Probably I am.

**Comment by Midnighter** [comment]
I can't remember now, does gzip (`gzip -c > ${prefix}.fastp.fastq.gz`) insert a timestamp in streaming mode? If so, it's better to avoid timestamps for reproducible outputs.

**Comment by jfy133** [comment]
I'm not sure, but we have a new module incoming so it's not important for now I think :)

**Pull request merged**